data/data/azure/dns/dns.tf

```diff
@@ -0,0 +1,28 @@
+resource "azurerm_dns_zone" "private" {
+  name                           = "${var.base_domain}"
```

Comment:
making the private dns zone to `basedomain` means that all clusters with same base domain but different cluster name cannot talk to each other as services inside this vnet assume that this private zone delegates all records `*.base_domain`

is this a wrong assessment?

Reply:
will change when using provided public zone

Reply:
fixed in latest